test/ui/viewJobRoleTests.ts

```diff
@@ -0,0 +1,98 @@
+import { Builder, WebDriver } from 'selenium-webdriver';
+import webdriver from 'selenium-webdriver';
+import { expect } from 'chai';
+import * as chrome from 'selenium-webdriver/chrome';
+
+describe('Job Roles Test', function () {
+
+    let driver: WebDriver;
+
+    before(async function () {
```

Comment:
Can we create a seperate class used for initialising the driver. This should be held in a separate class following the Page Object Model setup